relay-metrics/src/aggregation.rs

```diff
@@ -1041,12 +1039,17 @@ impl AggregatorService {
         I: IntoIterator<Item = Bucket>,
     {
         for bucket in buckets.into_iter() {
-            let tag = metric_name_tag(&self.name);
```

Comment:
I accidentally passed the aggregator name instead of the bucket name here, so the previous tag was useless (defaulted to "other").